### PR TITLE
showing bot command while using help solved 

### LINKED ISSUE #20: (bug) bot commands don't show up correctly in the help message
## Describe the bug
Possibly the commands are being rendered as markdown and thus the `_` (underscore) in the bot username makes text italic. 

## To Reproduce
Send the bot `/help@bot_name` and it will reply you with the broken help message.

## Expected behavior
The commands in the message should look like `/xkcd@bot_name`, `/links@bot_name` .... 

## Screenshots
jodc_bot_broken_help_msg
**Notice how the commands look like _/command@botname_**

## Platform
 OS agnostic

## Additional context
If its your first time contributing to the project, welcome :hugs: . Take a look at the [contributing guide](https://github.com/JIITODC/bot/blob/master/CONTRIBUTING.md) which is a step by step guide on getting the bot setup. 
We use the `python-telegram-bot` API wrapper. Here is a link to its documentation: https://python-telegram-bot.readthedocs.io/en/stable/index.html .
Do comment below something like 'I want to work on this issue' to let others know that you're working on this. Feel free to ask about any doubts you have or problems you're facing.

### PR DESCRIPTION
**Pull request by sarthakpant772**
### help.py
 to sove markdown of  ```_``` i used replace and replaced it with ```\_```.
and  instead of ParseMode.MARKDOWN i have used ParseMode.HTML,

## Does this resolve any currently open issues?
Resolves #20 

## Screenshot/GIF showing resolved bug/enhancement (if applicable)
..
image

### CODE REVIEW

File: commands/help.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import os
 
 bot_name = os.getenv("bot_name")
+bot_name.replace("_","\\_")
```
Comment (aayushmau5) [inline commands/help.py]: Now that the parse mode is in HTML, do we need to escape this `_`?

Comment (sarthakpant772) [reply commands/help.py]: yes it was not needed

**Comment by aayushmau5** [comment]
@sarthakpant772 I have some concerns with using HTML as the message parser.

Does the change you made affects other commands as well? 

Like for example, `/meetup` uses markdown as `[link](https://...)`, this will break cause it's in markdown.

**Comment by sarthakpant772** [comment]
The meet up thing is comming like 
image
is this how should it look?

**Comment by DelusionalOptimist** [comment]
> Does the change you made affects other commands as well?
>
> Like for example, /meetup uses markdown as [link](https://...), this will break cause it's in markdown.

I don't think it should affect other commands since we're only changing it for `/help` message. :thinking:

**Comment by aayushmau5** [comment]
Oh, you are right. It's only affecting the `/help` command. Then, this PR is good to go.

**Review by DelusionalOptimist — APPROVED** [review]
Looks good to me :heavy_check_mark:.
Thanks @sarthakpant772 :rocket:

**Review by aayushmau5 — APPROVED** [review]
Thanks @sarthakpant772 :rocket:

**Comment by DelusionalOptimist** [comment]
:tada:

**Pull request merged**